diff --git a/src/google/protobuf/compiler/cpp/cpp_message.cc b/src/google/protobuf/compiler/cpp/cpp_message.cc
index 6391643..378c11c 100644
--- a/src/google/protobuf/compiler/cpp/cpp_message.cc
+++ b/src/google/protobuf/compiler/cpp/cpp_message.cc
@@ -1086,7 +1086,7 @@ void MessageGenerator::GenerateClassDefinition(io::Printer* printer) {
       "  return reinterpret_cast<const $classname$*>(\n"
       "             &_$classname$_default_instance_);\n"
       "}\n"
-      "static constexpr int kIndexInFileMessages =\n"
+      "static int const kIndexInFileMessages =\n"
       "  $1$;\n"
       "\n", index_in_file_messages_);
 
diff --git a/src/google/protobuf/extension_set.h b/src/google/protobuf/extension_set.h
index e5ae3d4..ae253f2 100644
--- a/src/google/protobuf/extension_set.h
+++ b/src/google/protobuf/extension_set.h
@@ -656,7 +656,7 @@ class PROTOBUF_EXPORT ExtensionSet {
   // Grows the flat_capacity_.
   // If flat_capacity_ > kMaximumFlatCapacity, converts to LargeMap.
   void GrowCapacity(size_t minimum_new_capacity);
-  static constexpr uint16 kMaximumFlatCapacity = 256;
+  static uint16 const kMaximumFlatCapacity = 256;
   bool is_large() const { return flat_capacity_ > kMaximumFlatCapacity; }
 
   // Removes a key from the ExtensionSet.
-- 
2.17.1.windows.2

